app/views/controllers/admin/banners/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1><%= :change_banner_title.t %></h1>
+
+<%= form_with model: @banner, url: admin_banners_path, local: true, method: :post do |f| %>
+  <div>
+    <%= f.label :message %><br>
+    <%= f.text_area :message, rows: 5, cols: 120 %>
+  </div>
+  <%= f.submit :banner_update.t %>
```

Comment:
No div needed - please do use the MO form helpers here, that will help keep all our markup consistent during Bootstrap upgrades. 
```suggestion
  <%= text_area_with_label(form: f, field: :message, rows: 5) %>
  <%= submit_button(form: f, button: :banner_update.t) %>
```
Also, our Ruby style is parens for method calls, which I'd prefer be consistently used also in ERB for legibility.